Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -125,11 +145,25 @@ ActsExamples::ProcessCode ActsExamples::TrackParamsEstimationAlgorithm::execute(
       double charge = std::copysign(1, params[Acts::eBoundQOverP]);
       trackParameters.emplace_back(surface->getSharedPtr(), params, charge,
                                    m_covariance);
+      if (outputSeeds) {
+        outputSeeds->push_back(seed);
+      }
+      if (outputTracks && inputTracks != nullptr) {
+        outputTracks->push_back(inputTracks->at(iseed));
```

Comment:
```suggestion
        outputTracks->push_back((*inputTracks)[iseed]);
```

kinda ugly - feel free to close